plugins/activecontrols/trumbowyg.activecontrols.js

```diff
@@ -0,0 +1,85 @@
+/* ===========================================================
+ * trumbowyg.activecontrols.js v1.0
+ * Plugin to display active formattig buttons in buttons pane
+ * http://alex-d.github.com/Trumbowyg
+ * ===========================================================
+ * Author : Rastislav Švarba (ra100)
+ *          Twitter : @ra100
+ *          Website : ra100.net
+ */
+
+(function ($) {
+    'use strict';
+    $.extend(true, $.trumbowyg, {
+        opts: {
+            on: {
+                activecontrols: {
+                    events: 'mouseup keydown',
+                    handler: function (event, t) {
+                        t.saveSelection();
+                        if (t.selection !== null) {
+                            t.o.controlButtonActivate(t.selection.commonAncestorContainer.parentNode, t);
+                        }
+                    }
+                }
+            },
+            activeTags: [],
+            // Active tags
+            controlButtonActivate: function (element, t) {
+                var tagToButton = {
```

Comment:
maybe add that to $.trumbowyg object to made other plugins able to add others tags for other buttons